examples/lib/stories/parallax/parallax.dart

```diff
@@ -19,6 +20,11 @@ void addParallaxStories(Dashbook dashbook) {
       (_) => GameWidget(game: ComponentParallaxGame()),
       codeLink: baseLink('parallax/component.dart'),
     )
+    ..add(
+      'Non-fullscreen',
+      (_) => GameWidget(game: SmallParallaxGame()),
+      codeLink: baseLink('parallax/small_parallax.dart'),
```

Comment:
Maybe we could add some info about this example.

Since this is a very simple example, maybe just a small sentence would be enough. wdyt?

Reply:
Sounds good, just forgot about it.